src/interfaces.js

```diff
@@ -67,3 +68,21 @@ interface DynamicComponentGetViewStrategy {
     */
     getViewStrategy(): string|ViewStrategy;
 }
+
+/**
+* An optional interface describing the configuration object that can be specified to customize bindable properties.
+*/
+interface BindablePropertyConfig {
```

Comment:
it also takes a property name `defaultValue`, type any

Reply:
Thanks!